string label
cut_mission/Waypoint waypoint1
cut_mission/Waypoint waypoint2
